Controller/CategoryAdminController.php

```diff
@@ -40,8 +40,13 @@ public function listAction(Request $request = null)
         }
 
         $datagrid = $this->admin->getDatagrid();
+        $datagridValues = $datagrid->getValues();
 
-        if ($this->admin->getPersistentParameter('context')) {
+        $datagridContextIsSet = array_key_exists('context', $datagridValues) ?
+            !empty($datagridValues['context']['value']) : false;
```

Comment:
Maybe use `isset()` here then. BTW, why not simply use `isset()` directly, without the `array_key_exists` check?